unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AudioClip/AssetPromise_AudioClip.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace DCL
+{
+    public class AssetPromise_AudioClip : AssetPromise<Asset_AudioClip>
+    {
+        private readonly string url;
+        private readonly string id;
+        private readonly AudioType audioType;
+        private readonly IWebRequestController webRequestController;
+
+        private WebRequestAsyncOperation webRequestAsyncOperation;
+
+        public AssetPromise_AudioClip(string clipPath, ContentProvider provider) : this(clipPath, provider, Environment.i.platform.webRequest) { }
+
+        public AssetPromise_AudioClip(string clipPath, ContentProvider provider, IWebRequestController webRequestController)
+        {
+            this.url = provider.GetContentsUrl(clipPath);
+            this.id = this.url != null ? this.url.Substring(this.url.LastIndexOf('/') + 1) : $"{GetHashCode()}";
+            this.audioType = GetAudioTypeFromUrlName(clipPath);
+            this.webRequestController = webRequestController;
+        }
+
+        protected override void OnLoad(Action OnSuccess, Action OnFail)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                OnFail?.Invoke();
+                return;
+            }
+
+            webRequestAsyncOperation = webRequestController.GetAudioClip(url, audioType,
+                request =>
+                {
+                    bool supported = true;
+#if UNITY_EDITOR
+                    supported = audioType != AudioType.MPEG;
```

Comment:
```suggestion
                   
```
The `supported` logic shouldn't be there anymore, this was fixed by Mordi in another PR. The current version of unity works well with MPEG files.